R/data.R

```diff
@@ -115,7 +151,7 @@
 #' - `Grade 5`: Criteria defining lab value as Grade 5.
 #' - `Definition`: Holds the definition of the lab test abnormality.
 #' - `GRADE_CRITERIA_CODE`: variable to hold code that creates grade based on defined criteria.
-#' - `SI_UNIT_CHECK`: variable to hold unit of particular lab test. Used to check against input data
+#' - `UNIT_CHECK`: variable to hold SI unit of particular lab test. Used to check against input data
```

Comment:
Do we need to do anything for users since we changed a variable name?  Not sure how to handle this one? @bundfussr @manciniedoardo

Reply:
do we need a specific note about it in the changelog?

Reply:
not sure - can discuss later today

Reply:
Updated in NEWS Breaking changes section.